Sources/ff-ios-client-sdk/CfClient.swift

```diff
@@ -627,12 +627,34 @@ public class CfClient {
                     Logger.log("Message: \(decoded)")
                     
 					self.lastEventId = decoded.event
-					self.featureRepository.getEvaluationById(decoded.identifier ?? "", target: self.target.identifier, useCache: false, onCompletion: { (result) in
-						switch result {
-							case .failure(let error): onEvent(EventType.onEventListener(nil), error)
-							case .success(let evaluation): onEvent(EventType.onEventListener(evaluation), nil)
-						}
-					})
+                    
+                    // Handle Target Segmetn Events
+                    if decoded.event == "patch" && decoded.domain == "target-segment" {
```

Comment:
We can probably just remove the patch, and if its a target-segment domain take this action, otherwise we do the default for a flag.
```suggestion
                    if  decoded.domain == "target-segment" {
```

Reply:
I wrote the same comment today but I deleted it, because I am not sure do we want to respond on create event it will trigger every time for that env all flags evaluations and no flag wil be affected because new group is not in any rule